### DISCUSSION TITLE
How do you change the default bootstrap theme colors?

### DISCUSSION THREAD

**Discussion by luca992** [hfhbd/bootstrap-compose#186] [Q&A] [answered]
Just wondering, what would be the best way to change the default theme colors that are defined by bootstrap?

Found instructions for a normal project.
https://stackoverflow.com/questions/38792005/how-to-change-the-bootstrap-primary-color

But, I'm not sure how I can translate that to a kotlin project.

**Comment by hfhbd** [comment]
Yeah, this is still an open issue. You could either change them via `StyleSheet` in Kotlin only, or try to use Sass. 
https://kotlinlang.slack.com/archives/C01F2HV7868/p1621914625047700?thread_ts=1621797018.040900&cid=C01F2HV7868
I will create a sample later this day.

**Reply by hfhbd** [reply]
@luca992 I added a sample at https://github.com/hfhbd/bootstrap-compose/pull/195 which will be merged later this day.

**Comment by hfhbd** [comment, marked as answer]
I added it to the README and the showcase.
